ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -200,16 +200,19 @@ def test_torch_binary_cross_entropy(
     size_average=st.booleans(),
     reduce=st.booleans(),
     reduction=st.sampled_from(["mean", "none", "sum", None]),
-    dtype_and_pos_weight=helpers.array_or_none(
-        array_dtype="float",
-        min_value=0,
-        max_value=10,
-        allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
+    dtype_and_pos_weight=st.one_of(
+        helpers.dtype_and_values(
+            array_dtype="float",
```

Comment:
We should use `available_dtypes=helpers.get_dtypes("float")`, `array_dtypes` is not a parameter.